salt/output/pony.py

```diff
@@ -41,15 +43,20 @@
 
 # Import Python libs
 from __future__ import absolute_import
-import os
 import subprocess
 
 # Import Salt libs
+import salt.utils
 import salt.utils.locales
 
 
+__virtual_name__ = 'pony'
```

Comment:
Did we change this to ``__virtual_name__``. I've seen ``__virtualname__`` used elsewhere. The loader uses ``__virtualname__`` so I think this may need to be changed.